tools/Customization/DevHome.Customization/TelemetryEvents/SettingChangedEvent.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics.Tracing;
+using DevHome.Telemetry;
+using Microsoft.Diagnostics.Telemetry;
+using Microsoft.Diagnostics.Telemetry.Internal;
+
+namespace DevHome.Customization.TelemetryEvents;
+
+[EventData]
+public class SettingChangedEvent : EventBase
+{
+    public override PartA_PrivTags PartA_PrivTags => PrivTags.ProductAndServiceUsage;
+
+    public string Id
+    {
+        get;
+    }
+
+    public string Value
+    {
+        get;
+    }
+
+    public SettingChangedEvent(string id, string value)
+    {
+        Id = id;
+        Value = value;
+    }
+
+    public override void ReplaceSensitiveStrings(Func<string, string> replaceSensitiveStrings)
+    {
+        // No sensitive strings to replace.
+    }
+
+    public static void Log(string settingName, string value)
+    {
+        TelemetryFactory.Get<ITelemetry>().Log("Customization_SettingChanged_Event", LogLevel.Measure, new SettingChangedEvent(settingName, value));
```

Comment:
At this time I'm not sure any of these will come through, we've gotten approvals to use LogLevel.Critical around dev home. You do have to get approvals for each event, though.

Reply:
What is the recommended approach here? We will want to set up measures for these at some point but I'm not sure it's critical to do that until we have a good telemetry story for Windows customization. I could just change this to be Info for now.